sources/web/datalab/static.ts

```diff
@@ -228,6 +228,7 @@ function requestHandler(request: http.ServerRequest, response: http.ServerRespon
     'logo.png': 'datalab.png',
   };
 
+  response.setHeader('Cache-Control', 'public, max-age=3600');
```

Comment:
I think perhaps we want not to set this flag if we are running in LIVE mode, or maybe clear it in sendFile if isDynamic is set, around line 91.